### ISSUE TITLE
[Serious] Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#605]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87756 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Import/Export Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 12:49 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The contrast ratio between text and its background is not at least 4.5:1.
In Export Settings, select New, then select a few checkboxes, then navigate to the bottom and select Export.  If you select Copy to Clipboard, a green text status message appears that is low contrast.  See Import Export User Experience at 14:40.
Also occurs on Workflow, select a workflow to view, then Export, then Copy to Clipboard.

## Recommendation

Rule : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

How To Fix: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

Background: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

## Screenshots

![02_green white contrast](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87756/02_green_white_contrast.png)
